src/platform/site-wide/user-nav/components/HelpMenu.jsx

```diff
@@ -22,17 +22,13 @@ class HelpMenu extends React.Component {
             <a href={`${FACILITY_LOCATOR_URL}`}>Find a VA Location</a>
           </p>
           <p>
-            <a href="https://iris.custhelp.va.gov/app/ask ">
-              Submit a Help Request
-            </a>
+            <b>Call MyVA311:</b>
```

Comment:
I'm curious what @1Copenut thinks of this. It's not really a heading, more of a label, so should we use labelled-by or described-by?

Reply:
@1Copenut what do you think about this? This is how it is on vets, so I just replicated it.